[DOCS-7067] ACS Transform override case (#934)
Alfresco/docs-alfresco: content-services/7.2/develop/repo-ext-points/content-transformers-renditions.md

local.transform.pipeline.config.dir=shared/classes/alfresco/extension/transform/pipelines
```

> Local transform-core-aio docker image might need a [newer version of image and approach](#adding-pipelines-and-failover-transforms-to-a-t-engine)

Reviewer: Hey, sorry, I dont know whats going on here. Does this make sense? The link text doesnt make sense in the sentence does it? And I'm not sure about the use of Might.

It also links to a section that isnt called: newer version of image and approach.

Cheers.

Author: Thanks, I understand how it doesn't quite make sense. This is a line directly from the developer for this special case. I can change the link text, and hopefully everyone will accept the changes, and it will be easier to understand from a user's perspective.

local.transform.pipeline.config.dir=shared/classes/alfresco/extension/transform/pipelines
```

> Direct override is only possible in 3.0.0 version of alfresco-transform-core and higher. To override local transform-core-aio T-Engine configuration, the docker image needs a newer version of the image and follow this approach providing configuration directly to the T-Engine: [Adding pipelines and failover transforms to a T-engine](#adding-pipelines-and-failover-transforms-to-a-t-engine).

Reviewer 2: The meaning is severely changed. The problem with transform-core-aio is that it loads the configuration of other t-engines in a static manner, while you can override the configuration of other specific t-engines (like tika, libreoffice), the transform-core-aio (all-in-one) in particular has issues with overrides due to how it loads configs of all t-engines grouped beneath it.

"To override local transform-core-aio T-Engine configuration, the docker image needs to be version 3.0.0 or higher and following approach of providing configuration directly to the T-Engine should be used:  [Adding pipelines and failover transforms to a T-engine](#adding-pipelines-and-failover-transforms-to-a-t-engine)."

That could be used, but I wouldn't mention alfresco-transform-core here, it's a base project for all t-engines, shared file store, t-router, where there is no issue (albeit it's way harder than the new approach).

> Direct override is only possible in 3.0.0 version of alfresco-transform-core and higher. To override local transform-core-aio T-Engine configuration, the docker image needs a newer version of the image and follow this approach providing configuration directly to the T-Engine: [Adding pipelines and failover transforms to a T-engine](#adding-pipelines-and-failover-transforms-to-a-t-engine).

Reviewer: @Amy-Hyland  I think this makes more sense. What do you think?
Direct override is only possible in the 3.0.0 version of alfresco-transform-core and above. To override the local transform-core-aio T-Engine configuration, the docker image must use a newer version of the image. Follow this approach and provide the configuration directly to the T-Engine: [Adding pipelines and failover transforms to a T-engine](#adding-pipelines-and-failover-transforms-to-a-t-engine).

Author: @mstrankowski Does Oliver's suggestion still convey the technical details correctly? Can I incorporate his changes?

* `transformOptions` - A list of references to options required by
the pipeline transformer.

### Adding pipelines and failover transforms to a T-Engine

Since Content Services version 7.2 and ATS 1.5:

So far we have talked about defining pipelines and failover transforms in the Repository or ATS router pipeline files. It is also possible to add them to a T-Engine's configuration, even when they reference a transformer provided by another T-Engine. It is only when all transform steps exist that the pipeline or failover transform becomes available. Warning messages will be issued if step transforms do not exist.

Generally it is better to add them to T-Engines to avoid having to add an identical entry to both the Repository and ATS Router pipeline files.

> Since Alfresco Transform Core 3.0.0, config files with additional pipelines or overrides can be provided directly to a T-Engine image through additional config files (which may be resources on the classpath or external files) that are specified with Spring Boot properties such as `transform.config.file.<filename>` or environment variables like `TRANSFORM_CONFIG_FILE_<filename>`. In case of issues with changing configuration of transform-core-aio T-Engine such approach may be necessary.

Reviewer: ```suggestion
Since Alfresco Transform Core 3.0.0, config files with additional pipelines or overrides can be provided directly to a T-Engine image through additional config files (which may be resources on the classpath or external files) that are specified with Spring Boot properties such as `transform.config.file.<filename>` or environment variables like `TRANSFORM_CONFIG_FILE_<filename>`. You can use this approach if you have issues with changing the configuration of the transform-core-aio T-Engine.
```
